client/gutenberg/editor/calypsoify-iframe.tsx

```diff
@@ -147,7 +148,10 @@ class CalypsoifyIframe extends Component< Props & ConnectedProps & ProtectedForm
 			this.iframeRef.current.contentWindow
 		) {
 			this.successfulIframeLoad = true;
-			const { port1: iframePortObject, port2: transferredPortObject } = new MessageChannel();
+			const {
+				port1: iframePortObject,
+				port2: transferredPortObject,
+			} = new MessageChannel(); /* eslint-disable-line no-undef */
```

Comment:
The linter wants to say that we should write `window.MessageChannel` explicitly. 😉

Reply:
I couldn't figure out the eslint warning - thanks for the tip! Addressed in 450b822.